plugins/tableselection/plugin.js

```diff
@@ -251,7 +252,7 @@
 
 		if ( cells.length > 0 ) {
 			editor.editable().addClass( fakeSelectedEditorClass );
-			cells[ 0 ].getAscendant( 'table' ).addClass( fakeSelectedTableClass );
+			cells[ 0 ].getAscendant( 'table' ).data( fakeSelectedTableDatasetClass, fakeSelectedTableClass );
```

Comment:
I used empty string. Setting `true` just parse it to string, so it reflects with result of attribute: `data-cke-table-faked-selection-table="true"`.
Unfortunately method `data` is combination of setter and getter, so I cannot omit 2nd attribute when I want to use it as a setter.
Current approach seems to be most convenient. Another option would be design some simple general purpose data attribute, like `data-cke-class` and assign class values, but it might be confusing.